guides/plugins/plugins/storefront/remove-unnecessary-js-plugin.md

```diff
@@ -0,0 +1,40 @@
+# Remove unnecessary Javascript plugin
```

Comment:
Tbh I'm not sure about the "unnecessary". To me it sounds a bit have you have to remove a plugin which was unnecessary in the first place. Maybe just "Remove a Javascript plugin"